code/game/objects/items/rogueweapons/melee/flail.dm

```diff
@@ -183,6 +183,23 @@
 			if("onbelt")
 				return list("shrink" = 0.3,"sx" = -2,"sy" = -5,"nx" = 4,"ny" = -5,"wx" = 0,"wy" = -5,"ex" = 2,"ey" = -5,"nturn" = 0,"sturn" = 0,"wturn" = 0,"eturn" = 0,"nflip" = 0,"sflip" = 0,"wflip" = 0,"eflip" = 0,"northabove" = 0,"southabove" = 1,"eastabove" = 1,"westabove" = 0)
 
+//Ranged mace-like mode
+/datum/intent/whip/crack/blunt
+	name = "bludgen"
+	blade_class = BCLASS_BLUNT
+	recovery = 6
+	penfactor = 0	//Blunt weapon
+	reach = 2	//Less range than a normal whip by 1
+	icon_state = "instrike"
+	item_d_type = "blunt"
```

Comment:
With this flag it'll always have -100 AP (BLUNT_DEFAULT_PENFACTOR), so you can give it higher damfactor to your preference (or change it to another d_type if you'd prefer it worked with normal pen)

Reply:
Updated